Additions NFTS Pallet Page

filippoweb3 opened the pull request:
Continuation of PR #4486

Commit:
Added Jegor's feedback

Commit:
moved banner to roles

jsidorenko commented on docs/learn/learn-nft-pallets.md:
```diff
@@ -40,36 +40,34 @@ Setting up a collection implies different roles with different permissions:
+
+The user can decide to lock an item or collection’s metadata, attributes, and settings. Also, a
+locking mechanism can prevent unauthorized and unprivileged transfers (unprivileged actions can be
+re-allowed anytime).
```
Let's remove that paragraph

VadimSaveljev commented on docs/learn/learn-nft-pallets.md:
```diff
@@ -115,13 +113,6 @@ Everything is unlocked by default (bitflag value `0`).
-
 With all these options, one can decide to modify the price of the collection's items and who can
 mint, receive or buy items in that collection. Time constraints are available with `startBlock` and
 `endBlock` parameters. It is thus possible, for example, to create a schedule in which holders of
```
[for free is not correct](https://github.com/w3f/polkadot-wiki/pull/4585/files/1a5c30649e449cf86898501f14ce4dfd847d6c95#diff-b15eb6ac76682e89b9d092ab5e20d7a1cea4a7e88207918c4b57735e3ef536c4R119)
it could have a price + transaction fees always apply

filippoweb3 replied:
specified at the end of the paragraph

Commit:
Addressed Jegor's and Vadim's feedback

filippoweb3 commented:
@jsidorenko @SaltyCucumber I addressed your feedback, thank you! Do you think there will be additional things to add anytime soon? if yes, better leave this PR open.

jsidorenko commented:
I think we can merge this PR, thanks for the changes!

VadimSaveljev commented on docs/learn/learn-nft-pallets.md:
```diff
@@ -115,19 +105,13 @@ Everything is unlocked by default (bitflag value `0`).
-items in collection A (`HolderOf` parameter) will be able to claim for free a limited number of NFTs
-from Collection X (`maxSupply` parameter) only within a specific time frame. You can modify the
-parameters so anyone can buy more NFTs from Collection X.
+items in collection A (`HolderOf` parameter) will be able to claim a limited number of NFTs from
```
you are able to mint exact number of NFTs you have in Collection A. It's a one to one ratio.
So if you have 3 nfts in collection A, you'll be able to mint 3 nfts in collection X, each time you will "use" one of your nfts,  the said NFT will have an attribute that says it was already used, so you won't be able to mint with it anymore, but you can mint in another collection Y if it also uses collection A as a HolderOf.

VadimSaveljev commented on docs/learn/learn-nft-pallets.md:
```diff
@@ -115,19 +105,13 @@ Everything is unlocked by default (bitflag value `0`).
+items in collection A (`HolderOf` parameter) will be able to claim a limited number of NFTs from
+Collection X (`maxSupply` parameter) only within a specific time frame. You can modify the
+parameters so anyone can buy more NFTs from Collection X. Note that items are never free as even if
+they could not have a price, transaction fees always apply.
```
"To mint a NFT you'll need to pay the item price + transaction fee. Even if item is free, the transaction fee always apply."

Commit:
addressed Vadim's feedback

Commit:
grammar check on added paragraphs

filippoweb3 committed:
Merge branch 'master' into update-nft-pallet-info4
